server/davidia/models/messages.py

```diff
@@ -136,7 +178,7 @@ class HeatmapData(ImageData):
 
     domain: tuple[float, float]
     heatmap_scale: ScaleType = ScaleType.linear
-    colour_map: str
+    colour_map: Optional[ColourMap | Literal["Last"]] = None
```

Comment:
Add pydoc to explain semantics

Reply:
I've removed `Literal["Last"]` as it's now in ColourMap